README.md

```diff
@@ -140,11 +140,8 @@ There are additional optional variables that can be passed to the playbooks:
   - **admin_email**: The email address of the admin user\
     Default: `admin@email.com`
 
-  - **admin_id**: The launchpad id of the admin user, can optionally be left blank (`admin_id=`)\
-    Default: `admin`
-
-  - **import_ssh_keys**: Boolean representing whether we should import ssh_keys to MAAS\
-    Default: `true`
+  - **admin_id**: The launchpad id of the admin user\
+    Default: `''`
```

Comment:
My idea was to replace this with something like, optionally launchpad or github id of the user can be provided for import of ssh keys. (if we remove it below, it does not have a default value)